chat/chat-tool/tool/src/webapp/jsp/editRoom.jsp

```diff
@@ -83,9 +83,9 @@
             <h:panelGroup styleClass="chat-date-instructions">
               <sakai:instruction_message value="#{msgs.custom_date_instructions}" />
             </h:panelGroup>
-			<h:panelGroup styleClass="longtext indnt1 ">
-				<h:message for="startDate" styleClass="alertMessageInline" style="padding-top:2px;display:block;"><br></h:message>
-				<h:message for="endDate" styleClass="alertMessageInline" style="padding-top:2px;display:block;"><br/></h:message>
+			<h:panelGroup styleClass="date-validation longtext indnt1 ">
+			  <h:message for="startDate" styleClass="alertMessageInline" style="padding-top:2px;display:block;"><br></h:message>
+			  <h:message for="endDate" styleClass="alertMessageInline" style="padding-top:2px;display:block;"><br/></h:message>
```

Comment:
Spacing is still different here. Also, while you're in the file it's an opportunity to fix the issue of the CSS file not being loaded properly, it's likely just a path problem.

Better to fix problems while you're in the file, rather than ignoring them and proliferating frowned upon coding practises.